### PR TITLE
TTV: Wizard- credentials alert

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
Add an alert to notify the user to copy its credentials because they will never see them again

### CODE REVIEW

File: webui/src/pages/repositories/wizard/spark_config_step.jsx [hunk 1]
```diff
@@ -44,6 +45,9 @@ export const SparkConfigStep = ({onComplete=()=>{}}) => {
     }
     return (
         <Box sx={{display: 'flex', flexDirection: 'column', justifyContent: 'space-between'}}>
+            <Box sx={{width: '100%'}}>
+                <Alert variant="warning" className="mt-3">Copy the Secret Access Key and store it somewhere safe. You will not be able to access it again.</Alert>
```
Comment (talSofer) [inline webui/src/pages/repositories/wizard/spark_config_step.jsx:49]: nit; Consider keeping the text more consistent with https://github.com/treeverse/lakeFS/blob/33530b18aa599debe96cbb697f8da5aa5960f59d/webui/src/pages/setup/index.jsx#L73

**Review by talSofer — APPROVED** [review]
LGTM!

File: webui/src/pages/repositories/wizard/spark_config_step.jsx (hunk 1 above)
Comment (johnnyaug) [reply webui/src/pages/repositories/wizard/spark_config_step.jsx:49]: If we make them consistent I vote for the new version.
Would change "Secret Access Key" to "credentials" though.

**Review by johnnyaug — APPROVED** [review]

**Comment by johnnyaug** [comment]
Solves: #3410

**Pull request merged**